kustomize/commands/internal/util/util.go

```diff
@@ -42,7 +42,7 @@ func GlobPatternsWithLoader(fSys filesys.FileSystem, ldr ifc.Loader, patterns []
 		if len(files) == 0 {
 			loader, err := ldr.New(pattern)
 			if err != nil {
-				log.Printf("%s has no match", pattern)
+				return nil, fmt.Errorf("%s has no match: %w", pattern, err)
```

Comment:
I add test cases at https://github.com/kubernetes-sigs/kustomize/pull/5030/commits/85a4c4b82efaf0066b3026d6e96236c4caf299d6.

I think this function is called by only below 3 points.
https://github.com/search?q=repo%3Akubernetes-sigs%2Fkustomize+util.GlobPatternsWithLoader&type=code

Reply:
Can you update the comment description of this function to specify that it returns an error if there are no matching files and it can't load from remote?